habitat-baselines/habitat_baselines/rl/ddppo/algo/ddppo.py

```diff
@@ -23,16 +23,30 @@ def _recursive_apply(inp, fn):
         return fn(inp)
 
 
+def _convert_to_numpy_safe(t):
```

Comment:
typing, is t supposed to be numpy array or torch tensor. 
If it is supposed to be a torch tensor, then it should not return t if it is on GPU

Reply:
Addressed with docstring and typing.